src/Tabs/TabGroup.test.js

```diff
@@ -89,4 +89,27 @@ describe('<Tabs />', () => {
             ).toBe('Sample');
         });
     });
+
+    describe('null tab children', () => {
+        test('should render all tabs that are not null', () => {
+            const tabsWithNull = (
+                <TabGroup>
+                    <Tab
+                        id='1'
+                        title='Tab 1'>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore et ducimus veritatis officiis amet? Vitae officia optio dolor exercitationem incidunt magnam non, suscipit, illo quisquam numquam fugiat? Debitis, delectus sequi?
+                    </Tab>
+                    {null}
```

Comment:
Minor nit here: how about writing this more like:
```js
const condition = false;

...

    {condition && <Tab id='2' title='Tab 2'>Lorem ipsum</Tab>}
```
This will ultimately pass in `null`, but it makes the test look more "real".